stub/UVFunctions.php

```diff
@@ -0,0 +1,2039 @@
+<?php
+
+/**
+ * create a `new` loop handle.
+ *
+ * @return UVLoop
+ */
+function uv_loop_new()
+{
+}
+
+/**
+ * Returns the initialized default loop.
+ * It may return NULL in case of allocation failure. This function is just a convenient
+ * way for having a global loop throughout an application, the default loop is in no way
+ * different than the ones initialized with `uv_loop_new()`.
+ *
+ * @return UVLoop
+ */
+function uv_default_loop()
+{
+}
+
+/**
+ * This function runs the event loop. It will act differently depending on the
+ * specified `$mode`.
+ *
+ * @param UVLoop $loop
+ * @param int $mode
+ *  - `UV::RUN_DEFAULT`: Runs the event loop until the reference count drops to
+ *    zero. Always returns zero.
+ *  - `UV::RUN_ONCE`: Poll for new events once. Note that this function blocks if
+ *    there are no pending events. Returns zero when done (no active handles
+ *    or requests left), or non-zero if more events are expected (meaning you
+ *    should run the event loop again sometime in the future).
+ *  - `UV::RUN_NOWAIT`: Poll for new events once but don't block if there are no
+ *    pending events.
+ */
+function uv_run(UVLoop $loop = null, int $mode = UV::RUN_DEFAULT)
+{
+}
+
+/**
+ * start polling.
+ *
+ * If you want to use a socket. please use uv_poll_init_socket instead of this. Windows can't handle socket with this function.
+ *
+ * @param UVPoll $poll
+ * @param int $events UV::READABLE and UV::WRITABLE flags.
+ * @param callable $callback expects (UVPoll $poll, int $status, int $events, mixed $connection)
```

Comment:
mixed? can we use the actual union?

Reply:
yes, and i see you have `resource $fd` in proto, will fixed.